src/charm.py

```diff
@@ -41,7 +42,11 @@ def __init__(self, *args):
         self.logger = logging.getLogger(__name__)
         self._port = self.model.config["mlflow_port"]
         self._container_name = "mlflow-server"
+        self._database_name = "mlflow"
```

Comment:
Is it possible to make this configurable?

Reply:
Tbh I dont think there is usecase for different db name ... I am not sure ... Last time I was instructed not to overconfigure the charms :)

Reply:
agree with Michal.
Let's keep DB name solid

Reply:
Sounds good. I will also hard code it in mine, to keep uniform.
The only usecase I can think of is customer having single MySQL installation with different DBs (different DB names) and related to different MLflow charms OR already having DB with name `mlflow` in MySQL DB.
There is an issue with hardcoded S3 bucket naming in some of our other charms So, same goes for DB names.